This is a test for #658: a kernel panic seen when patching an exported
function (e.g., kmalloc) which is used by patch_init().
---
diff --git a/mm/slub.c b/mm/slub.c
index 067598a..b517aec 100644
--- a/mm/slub.c
+++ b/mm/slub.c
@@ -3719,6 +3719,9 @@ void *__kmalloc(size_t size, gfp_t flags)
 	struct kmem_cache *s;
 	void *ret;
 
+	if (!jiffies)
+		printk("kpatch kmalloc\n");
+
 	if (unlikely(size > KMALLOC_MAX_CACHE_SIZE))
 		return kmalloc_large(size, flags);
 
